packages/vc-http-api-test-server/__fixtures__/verifiableCredentials/case-29.json

```diff
@@ -19,7 +19,7 @@
     "modeOfTransport": "Truck",
     "portOfEntry": {
       "type": "ShippingStop",
-      "shippingStopAddress": {
+      "https://schema.org/PostalAddress": {
```

Comment:
hmm, this does not look correct.

Reply:
This change enables verification to succeed. Should something else be done?
`shippingStopAddress` is no longer defined in the latest context file, and vocab is used, so the term expands to `https://w3id.org/traceability/#undefinedTermshippingStopAddress`

Reply:
> This change enables verification to succeed. Should something else be done?

yes, we should make the credential both look correct and verify.... we don't want to start seeing credentials with IRIs floating around in them.

There have been a huge number of changes to supply chain context since these cases were last created, I am not sure that even keeping them in this suite is the best move... we might be better off with a generic suite that had not industry specific contexts, and industry specific ones (which we have for trace already).

Reply:
Proposing removal in #18

Reply:
agree, remove the failing case... we should address specific interop flows in test suites that have more context.... I think its not reasonable to assume that APIs implementing this spec will have all contexts for all forms of credentials.